inject-groovy/src/main/groovy/io/micronaut/ast/groovy/visitor/GroovyClassElement.java

```diff
@@ -906,7 +906,7 @@ public Optional<MethodElement> getReadMethod() {
                     private String getGetterName(String propertyName, ClassElement type) {
                         return NameUtils.getterNameFor(
                                 propertyName,
-                                type.equals(PrimitiveElement.BOOLEAN) || type.getName().equals(Boolean.class.getName())
+                                type.equals(PrimitiveElement.BOOLEAN)
```

Comment:
Is this because what you mentioned yesterday about Groovy 4 no longer generating the `isField()` method for `Boolean` fields?

Reply:
Yes